api/CSSSupportsRule.json

```diff
@@ -5,57 +5,54 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSSupportsRule",
         "support": {
           "chrome": {
-            "version_added": false
+            "version_added": "41"
```

Comment:
This interface was originally not exposed, and I fixed that in https://chromium.googlesource.com/chromium/src/+/3e3b38c7fdf9a0753b369576713578aa0895262c, which was M41 per https://www.chromium.org/developers/calendar.

Testing http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8599 in Chrome 40 will confirm earlier support. Can you add a custom test for this? We should mark the initial range as `partial_implementation`.